app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :likes
+  has_many :comments
```

Comment:
- [ ] These associations are very correct. But, according to the ERD we need to refactor them. Do you remember the story about the foreign keys being **_`AuthorId`_** in all tables? In this case, these associations should follow the example below. Please refactor all three.

`has_many :posts, foreign_key: 'author_id'`

Reply:
Well solved